Lib/functools.py

```diff
@@ -274,7 +274,7 @@ def reduce(function, sequence, initial=_initial_missing):
 ################################################################################
 
 
-class __PlaceholderTypeBase:
+class PlaceholderTypeBase:
```

Comment:
Since the reference is removed, the intermediate class is seems unnecessary:

```python
class PlaceholderType:
    __instance = None
    __slots__ = ()
    def __init_subclass__(cls, *args, **kwargs): ...
    ...

Placeholder = PlaceholderType()
del PlaceholderType
```